src/zip/fs.ts

```diff
@@ -66,6 +74,7 @@ export interface ZipOptions<TBuffer extends ArrayBufferLike = ArrayBuffer> {
 export class ZipFS<TBuffer extends ArrayBufferLike = ArrayBuffer> extends Readonly(FileSystem) {
 	protected files: Map<string, FileEntry<TBuffer>> = new Map();
 	protected directories: Map<string, Set<string>> = new Map();
+	protected folded: Map<string, string> = new Map();
```

Comment:
I think I understand why we need this but can you walk me through it just to make sure?

Reply:
The idea is to keep the original case (the one used by the user) in a map, and when the files/folders are listed the user will see the original name in the original case.

Reply:
Okay, thanks.